integration-tests/jdbc/pom.xml

```diff
@@ -40,6 +40,14 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-log</artifactId>
         </dependency>
+        <dependency>
```

Comment:
`camel-quarkus-bean-deployment` and `camel-quarkus-timer-deployment` dependencies need adding into the `virtualDependencies` profile.

Reply:
A simple `mvn process-resources -Pformat -N` from root directory should format this automatically.